datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1490,6 +1490,8 @@ pub struct Union {
 pub struct CreateMemoryTable {
     /// The table name
     pub name: OwnedTableReference,
+    /// The ordered list of columns in the primary key, or an empty vector if none
+    pub primary_key: Vec<Column>,
```

Comment:
TODO: asc, desc

Reply:
I think we might not be able to get the PK index ordering from AST.

Reply:
I did some research: sqlparser is correct in this case, as `CreateTable` has columns that define the `unique constraint` that is the `primary key`. Constraints don't have ordering, indexes do. 

I assume that if the users does a `create index` we will receive information about sort order of each column, but either way it's proper not to have it here.